### PR TITLE
Atom 8763 success responses

### PR DESCRIPTION
**Pull request by dougperez**

### CODE REVIEW

File: lib/conduit/fortress/parsers/base.rb [hunk 1]
```diff
@@ -39,11 +39,15 @@ def error_messaage
...
       def response_errors
-        return [] if error_messaage.empty?
+        return [] unless has_errors? 
 
         error = { message: error_messaage }
         [Conduit::Error.new(error)]
       end
+
+      def has_errors?
+        error_messaage && error_messaage != "0"
```
Comment (pedromartinez) [inline lib/conduit/fortress/parsers/base.rb]: message is misspelt

Comment (dougperez) [reply lib/conduit/fortress/parsers/base.rb]: Fixed

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -4,9 +4,3 @@
 include Conduit::Driver::Fortress
 
 Dir[File.join(Dir.pwd, "spec/support/**/*.rb")].each { |file| require file }
-
-RSpec.configure do |config|
-  config.before(:suite) do
-    Excon.defaults[:mock] = true
```
Comment (pedromartinez) [inline spec/spec_helper.rb]: Did you mean to remove this?

Comment (dougperez) [reply spec/spec_helper.rb]: We do it in the mocking layer so doesn't seem like it needed here.

Comment (pedromartinez) [reply spec/spec_helper.rb]: I think this was to prevent anybody from using a live request via RSpec

Comment (dougperez) [reply spec/spec_helper.rb]: Ok

**Pull request merged**